src/routes/chat/Chat.container.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
```

Comment:
Avoid connect, use `import { useSelector, useDispatch } from 'react-redux'` in `Chat.jsx` itself instead.